docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -90,10 +89,37 @@ To update the metadata and playback information on the server side, and to retri
         Tizen.Log.info(LogTag, $"Playback state is {controller.GetPlaybackState()}");
         ```
 
-<a name="send_media"></a>
+## Updating and Retrieving Playlist
+
+To create a playlist from the server side, create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavorite");
+    ```
+
+You can also create playlist with Metadata using the following code:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavoriate",
+        new Dictionary<string, MediaControlMetadata>()
+        {
+            {"Song1", new MediaControlMetadata() {Author = "Someone"} },
+            {"Song2", new MediaControlMetadata()}
+        }
+    );
+    ```
+
+To retrieve the playlist and metadata information on the client side:
```

Comment:
@hsgwon 

**Suggestion:**
You may add "use the following code:" to this sentence, as added in the line no. 100 "You can also create playlist with Metadata using the following code:". It will give consistency to the page.

Reply:
Done